app/packages/core/src/components/Actions/GroupMediaVisibilityContainer.tsx

```diff
@@ -0,0 +1,94 @@
+import { PopoutSectionTitle } from "@fiftyone/components";
+import ViewComfyIcon from "@mui/icons-material/ViewComfy";
+import React, { useRef, useState } from "react";
+import useMeasure from "react-use-measure";
+
+import {
+  groupMediaIs3DVisible,
+  groupMediaIsCarouselVisible,
+  groupMediaIsImageVisible,
+  useOutsideClick,
+} from "@fiftyone/state";
+import { useRecoilState } from "recoil";
+import styled from "styled-components";
+import Checkbox from "../Common/Checkbox";
+import style from "../Modal/Group.module.css";
+import { PillButton } from "../utils";
+import Popout from "./Popout";
+
+interface GroupMediaVisibilityProps {
+  modal: boolean;
+}
+
+const TITLE = "Toggle Media";
+
+const Container = styled.div`
+  position: relative;
+`;
+
+const GroupMediaVisibilityPopout = ({
+  modal,
+  bounds,
+}: {
+  modal: boolean;
+  bounds: ReturnType<typeof useMeasure>[1];
```

Comment:
Noting that `bounds` is some tech debt of my own doing, I don't think it has any effect if you want to remove it.